kroxylicious-filters/kroxylicious-multitenant/src/test/java/io/kroxylicious/proxy/filter/multitenant/MultiTenantFilterTest.java

```diff
@@ -35,31 +39,42 @@
 
 import io.kroxylicious.proxy.filter.FilterAndInvoker;
 import io.kroxylicious.proxy.filter.FilterContext;
-import io.kroxylicious.proxy.filter.FilterFactoryContext;
 import io.kroxylicious.proxy.filter.FilterInvoker;
 import io.kroxylicious.proxy.filter.RequestFilterResult;
 import io.kroxylicious.proxy.filter.ResponseFilterResult;
 import io.kroxylicious.proxy.filter.multitenant.config.MultiTenantConfig;
+import io.kroxylicious.test.condition.kafka.FetchResponseDataCondition;
 import io.kroxylicious.test.requestresponsetestdef.ApiMessageTestDef;
 import io.kroxylicious.test.requestresponsetestdef.RequestResponseTestDef;
 
 import static com.google.common.base.Preconditions.checkState;
 import static com.google.common.collect.Iterables.getOnlyElement;
+import static io.kroxylicious.test.condition.kafka.ApiMessageCondition.forApiKey;
+import static io.kroxylicious.test.condition.kafka.ProduceRequestDataCondition.produceRequestMatching;
 import static io.kroxylicious.test.requestresponsetestdef.KafkaApiMessageConverter.requestConverterFor;
 import static io.kroxylicious.test.requestresponsetestdef.KafkaApiMessageConverter.responseConverterFor;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.assertArg;
 import static org.mockito.Mock.Strictness.LENIENT;
+import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 @ExtendWith(MockitoExtension.class)
-class MultiTenantTransformationFilterFactoryFilterTest {
+class MultiTenantFilterTest {
     private static final Pattern TEST_RESOURCE_FILTER = Pattern.compile(
-            String.format("%s/.*\\.yaml", MultiTenantTransformationFilterFactoryFilterTest.class.getPackageName().replace(".", "/")));
+            String.format("%s/.*\\.yaml", MultiTenantFilterTest.class.getPackageName().replace(".", "/")));
     private static final String TENANT_1 = "tenant1";
 
+    private static final String VIRTUAL_CLUSTER_NAME = "vc1";
+    private static final String TEST_TOPIC = "testTopic";
+    @Mock
+    private FilterContext filterContext;
+
     private static List<ResourceInfo> getTestResources() throws IOException {
-        var resources = ClassPath.from(MultiTenantTransformationFilterFactoryFilterTest.class.getClassLoader()).getResources().stream()
+        var resources = ClassPath.from(MultiTenantFilterTest.class.getClassLoader()).getResources().stream()
                 .filter(ri -> TEST_RESOURCE_FILTER.matcher(ri.getResourceName()).matches()).toList();
 
         // https://youtrack.jetbrains.com/issue/IDEA-315462: we've seen issues in IDEA in IntelliJ Workspace Model API mode where test resources
```

Comment:
Idea claim to have fixed https://youtrack.jetbrains.com/issue/IDEA-315462 (without leaving any comments on the issue)

Reply:
I'll test that out at some point.  Thank you for the ping.